### PR TITLE
feat: change default position to TOP for widgets with labels

### LINKED ISSUE #17143: [Feature]: Change default label position to top for all widgets which have labels
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

Change default label position to top for all widgets which have labels and increase their height so that each widget with label fits in the bounding box. 

Widgets to be updated
1. All inputs
2. All selects
3. Datepicker
4. Group widgets

No need to do it in checkbox, switch and rich text editor

### Why should this be worked on?

When user drops the widget in the form/container the label cuts off today which then means user has to open the property pane and adjust the label there

### PR DESCRIPTION
**Pull request by adarshlilha**
## Description

Changes default label position to TOP for the below widgets:
- Inputs
    - Text Input
    - Phone Input
    - Currency Input
- Select
    - Select
    - TreeSelect
    - MultiSelect
    - MultiTree Select
- DatePicker
- Group Widgets
    - Radio group
    - Switch group
    - Checkbox group

Fixes #17143 
Screenshot 2022-10-19 at 19 00 09

## Type of change
- New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

- Manual - does not break the existing tests

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (mohanarpit) [inline yarn.lock:1]: This diff doesn't look right. Can we please remove this file from the PR?

**Comment by Tooluloope** [comment]
Screenshot 2022-10-25 at 22 38 43
The deploy preview seems not to work

File: yarn.lock (hunk 1 above)
Comment (Tooluloope) [reply yarn.lock:1]: Please fix this file and pull latest release

**Review by Tooluloope — CHANGES_REQUESTED** [review]

**Review by Tooluloope — APPROVED** [review]

File: yarn.lock (hunk 1 above)
Comment (adarshlilha) [reply yarn.lock]: file removed :)

**Pull request merged**